### ISSUE TITLE
Cargo.toml crate completion doesn't work for target-specific dependencies

### ISSUE THREAD

**Issue by jonas-schievink** [intellij-rust/intellij-rust#2856]
## Environment

* **Intellij-Rust plugin version:** 0.2.0.2104-181
* **Rust toolchain version:** rustc 1.28.0 (9634041f0 2018-07-30)
* **IDE name and version:** IntelliJ IDEA 2018.1.5 (Community Edition)
* **Operating system:** Arch Linux

## Problem description

Completion of crates.io crates doesn't seem to work inside target-specific dependency keys like `[target.'cfg(windows)'.dependencies]`

## Steps to reproduce

Try typing crate names in this Cargo.toml:

```toml
[package]
name = "anything"
version = "0.1.0"
authors = ["Jonas Schievink <jonasschievink@gmail.com>"]

[dependencies]

[dev-dependencies]

[target.'cfg(windows)'.dependencies]

[target.'cfg(unix)'.dependencies]
```

The first 2 dependency keys (`dependencies` and `dev-dependencies`) work fine, the last 2 don't.

Presumably, this is caused by this line, which only looks at the first key (here, `target`) to determine whether completion should be done:

https://github.com/intellij-rust/intellij-rust/blob/6a24958d7dcd86b28edbfd0ad29da2f169fe564b/src/main/kotlin/org/rust/toml/RsTomlCompletionProvider.kt#L56

Instead of looking at the first element, it should probably check the *last* one.

**Comment by Undin** [comment]
Thanks for the report!
We know about it and it is already fixed in #2647

**Comment by jonas-schievink** [comment]
Great, thanks!
